src/components/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import '../style/App.css';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.status = {
+      result: '0',
+    };
+  }
+
+  render() {
+    const { result } = this.status;
+    return (
+      <div className="App">
+        <Display result={result} />
```

Comment:
Also not needed for this milestone.
```suggestion
        <Display />
```